web/tests/input_pill.test.js

```diff
@@ -674,5 +669,24 @@ run_test("getCurrentText/onTextInputHook", ({mock_template}) => {
         preventDefault: noop,
     });
 
-    assert.deepEqual(widget.items(), [items.blue, items.red]);
+    assert.deepEqual(widget.items(), [items.blue, items.red, items.yellow]);
+});
+
+run_test("onTextInputHook", () => {
+    const info = set_up();
+    const config = info.config;
+    const widget = input_pill.create(config);
+    const $container = info.$container;
+
+    let hookCalled = false;
+    const onTextInputHook = () => {
+        hookCalled = true;
+    };
+
+    widget.onTextInputHook(onTextInputHook);
+
+    const input_handler = $container.get_on_handler("input", ".input");
+    input_handler();
+
+    assert.ok(hookCalled);
```

Comment:
This test seems pretty light of verifying anything -- you seem to be just getting coverage, not checking whether anything actually happened. Is that the intent?

I would have thought it worth at least validating the values of the input element, even if we're just checking that they aren't changed by this handler.

Reply:
My intent was to just check if the registered hook is getting called or not correctly by the `input` event handler defined. But yeah I think we can assert the current text value inside our hook to make sure that the hook always gets the updated value?